docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -879,9 +879,9 @@ spec:
 apiVersion: v1
 kind: policy
 spec:
-  name: No 1Password emergency kit stored on desktop or in downloads (macOS)
-  query: SELECT 1 WHERE NOT EXISTS (SELECT 1 FROM file WHERE filename LIKE '%Emergency Kit%.pdf' AND (path LIKE '/Users/%%/Desktop/%%' OR path LIKE '/Users/%%/Documents/%%' OR path LIKE '/Users/%%/Downloads/%%' OR path LIKE '/Users/Shared'));
-  description: "Looks for PDF files with file names typically used by 1Password for emergency recovery kits."
+  name: No 1Password emergency kit stored in desktop, documents, or downloads folders (macOS)
+  query: SELECT 1 WHERE NOT EXISTS (SELECT 1 FROM file WHERE filename LIKE '%Emergency Kit%.pdf' AND (path LIKE '/Users/%%/Desktop/%' OR path LIKE '/Users/%%/Documents/%' OR path LIKE '/Users/%%/Downloads/%' OR path LIKE '/Users/Shared/%'));
```

Comment:
I believe the original query was not correct.

`%%` is actually supported only at the end:

https://osquery.readthedocs.io/en/latest/deployment/file-integrity-monitoring/#matching-wildcard-rules
> Note: You cannot match recursively inside a path. For example /Users/%%/Configuration.conf is not a valid wildcard.

https://www.kolide.com/blog/the-file-table-osquery-s-secret-weapon
> Double wildcards can only ever be used at the end of a string eg. /foo/bar/%%